src/app/Instance/InstancePage/InstancePage.tsx

```diff
@@ -292,11 +292,7 @@ const InstancePage = (): JSX.Element => {
             activeKey={activeTabKey}
           >
             {activeTabKey === INSTANCE_PAGE_TAB_KEYS.processors && (
-              <ProcessorsTabContent
-                instanceId={instanceId}
-                pageTitle={getPageTitle(bridge)}
-                bridgeStatus={bridge?.status}
-              />
+              <BridgeOverview />
```

Comment:
Could you revert this and leave `ProcessorTabContent` here for the time being?
We will replace it with `BridgeOveriview` as soon as we plug it to the APIs.